tests/Seeds.ts

```diff
@@ -388,16 +399,29 @@ async function seed(): Promise<void> {
     MERCH_ITEM_1_OPTION_L,
     MERCH_ITEM_1_OPTION_XL,
   ];
+  MERCH_ITEM_1.photos = [
+    MERCH_ITEM_1_PHOTO_0,
+    MERCH_ITEM_1_PHOTO_1,
+    MERCH_ITEM_1_PHOTO_2,
+  ];
+
   const MERCH_ITEM_2 = MerchFactory.fakeItem({
     collection: MERCH_COLLECTION_1,
     itemName: 'Hack School Sticker Pack (4) - Cyan',
-    picture: 'https://i.imgur.com/pSZ921P.png',
     description: 'Make space on your laptop cover for these Cyan stickers. Pack of 4, size in inches.',
     monthlyLimit: 5,
     lifetimeLimit: 25,
     hidden: false,
     hasVariantsEnabled: true,
   });
+  const MERCH_ITEM_2_PHOTO_0 = MerchFactory.fakePhoto({
+    merchItem: MERCH_ITEM_2,
+  });
+  const MERCH_ITEM_2_PHOTO_1 = MerchFactory.fakePhoto({
+    merchItem: MERCH_ITEM_2,
+    picture: 'https://i.imgur.com/pSZ921P.png',
+    position: 1,
+  });
```

Comment:
Should photo_0 have a position and a picture?

Reply:
Similar to the comment above, I left it blank as it assume default values